x/ccv/provider/keeper/throttle.go

```diff
@@ -82,18 +91,29 @@ func (k Keeper) GetEffectiveValPower(ctx sdktypes.Context,
 func (k Keeper) HandlePacketDataForChain(ctx sdktypes.Context, consumerChainID string,
 	slashPacketHandler func(sdktypes.Context, string, ccvtypes.SlashPacketData),
 	vscMaturedPacketHandler func(sdktypes.Context, string, ccvtypes.VSCMaturedPacketData),
+	vscMaturedHandledThisBlock int,
 ) {
 	// Get slash packet data and trailing vsc matured packet data, handle it all.
 	slashFound, slashData, vscMaturedData, seqNums := k.GetSlashAndTrailingData(ctx, consumerChainID)
+	seqNumsHandled := []uint64{}
 	if slashFound {
 		slashPacketHandler(ctx, consumerChainID, slashData)
+		// Slash packet should always be the first packet in the queue, so we can safely append the first seqNum.
```

Comment:
Maybe link to parts of the code that make this more apparent.

Reply:
Good idea -> https://github.com/cosmos/interchain-security/commit/ba0d6abefa3b7179a49278d05580b494c617937c

Reply:
Thanks, resolving